include/libtorrent/kademlia/dht_storage.hpp

```diff
@@ -70,34 +70,24 @@ namespace dht
 		void reset();
 	};
 
-	struct TORRENT_EXPORT dht_immutable_item
+	struct TORRENT_EXPORT dht_immutable_data
```

Comment:
I will remove the `bloom_filter` field and I think that would give more credibility to the new structs. Naming can be an issue.